src/sync.py

```diff
@@ -1,74 +1,91 @@
-import copy
+import time
 
-from src.jira import JiraIssue
-from src.zenhub import ZenHubIssue
+from src.jira import JiraRepo
+from src.zenhub import ZenHubRepo
 
 
 class Sync:
+    """Assumptions:
+    - Unito has run currently, so that
+        - Each issue has a counterpart in each management system
+        - Each issue's description says the name of the issue it's linked with"""
+    @staticmethod
+    def sync_board(source: 'Board', sink: 'Board'):
+
+        if source.__class__.__name__ == 'ZenHubRepo' and sink.__class__.__name__ == 'JiraRepo':
+            for issue in source.issues.values():
+                Sync.sync_from_specified_source(issue, sink.issues[issue.jira_key])
+
+        elif source.__class__.__name__ == 'JiraRepo' and sink.__class__.__name__ == 'ZenHubRepo':
+            for issue in source.issues.values():
+                for i in range(3):  # Allow for 3 tries
```

Comment:
Is this a HTTP GET retry? Has it happened to you in the past that the server didn't respond the first time? If not, what try is it?

Reply:
Yeah, I have occasionally reached the api rate limit, which caused it to crash. I added that try block so that if an error happens, it tries again in 10 seconds, hopefully the api limit will have reset.